common/app/navigation/ReaderRevenueSite.scala

```diff
@@ -32,4 +32,8 @@ object ReaderRevenueSite extends enumeratum.Enum[ReaderRevenueSite] {
   case object SupportContribute extends ReaderRevenueSite {
     val url: String = s"${Configuration.id.supportUrl}/contribute"
   }
+
+  case object SupportGifting extends ReaderRevenueSite {
+    val url: String = s"${Configuration.id.supportUrl}/subscribe"
```

Comment:
why is this the same as SupportSubscribe?  I guess we don't have a gift landing page?
https://support.theguardian.com/subscribe

Reply:
Exactly that